lib/caseflow/service.rb

```diff
@@ -2,30 +2,30 @@
 
 require 'common/client/concerns/monitoring'
 
-module Appeals
+module Caseflow
   ##
-  # Proxy Service for Appeals Caseflow.
+  # Proxy Service for appeals in Caseflow.
   #
-  # @example Create a service and fetching appeals for a user
-  #   appeals_response = Appeals::Service.new.get_appeals(user)
+  # @example Create a service and fetching caseflow for a user
+  #   caseflow_response = Caseflow::Service.new.get_appeals(user)
   #
   class Service < Common::Client::Base
     include SentryLogging
     include Common::Client::Monitoring
 
-    configuration Appeals::Configuration
+    configuration Caseflow::Configuration
 
     STATSD_KEY_PREFIX = 'api.appeals'
```

Comment:
I think we have some specs that may test for this string

Reply:
Fixed those tests.